### PR TITLE
Initial codegen for securitycenter

### PR DESCRIPTION
**Pull request by dazuma**
Does not include kokoro configs. Will add those in a follow-up PR.

### CODE REVIEW

File: google-cloud-security_center/lib/google/cloud/security_center/v1/credentials.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+module Google
+  module Cloud
+    module SecurityCenter
+      module V1
+        class Credentials < Google::Auth::Credentials
+          SCOPE = [
+            "https://www.googleapis.com/auth/cloud-platform"
+          ].freeze
+          PATH_ENV_VARS = %w(SECURITYCENTER_CREDENTIALS
```
Comment (TheRoyalTnetennba) [inline google-cloud-security_center/lib/google/cloud/security_center/v1/credentials.rb]: Should this be `SECURITY_CENTER_CREDENTIALS`?

File: google-cloud-security_center/lib/google/cloud/security_center/v1/security_center_client.rb [hunk 2]
```diff
@@ -0,0 +1,1772 @@
...
+              :"organization" => organization,
+              :"source" => source,
+              :"finding" => finding
+            )
+          end
+
+          # Returns a fully-qualified organization resource name string.
+          # @param organization [String]
+          # @return [String]
+          def self.organization_path organization
```
Comment (TheRoyalTnetennba) [inline google-cloud-security_center/lib/google/cloud/security_center/v1/security_center_client.rb]: should this have the instance method aliases like https://github.com/googleapis/google-cloud-ruby/pull/3127 ?

File: google-cloud-security_center/lib/google/cloud/security_center/v1/credentials.rb (hunk 1 above)
Comment (dazuma) [reply google-cloud-security_center/lib/google/cloud/security_center/v1/credentials.rb]: Fixed.

File: google-cloud-security_center/lib/google/cloud/security_center/v1/security_center_client.rb (hunk 2 above)
Comment (dazuma) [reply google-cloud-security_center/lib/google/cloud/security_center/v1/security_center_client.rb]: Added

**Pull request merged**